html/webappapis/animation-frames/callback-handle.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>AnimationTiming Test: FrameRequestCallback - valid callback handle</title>
+<link rel="author" title="Intel" href="http://www.intel.com">
+<link rel="help" href="http://www.w3.org/TR/animation-timing/#Window-interface-extensions">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<div id="log"></div>
+<script>
+
+  async_test(t => {
+    let requestId = window.requestAnimationFrame(callback);
+
+    function callback(time) {
```

Comment:
This test would be much simpler as a sync test that just did the assert after calling requestAnimationFrame, and passed an empty function to rAF.